scss/book-list.scss

```diff
@@ -7,6 +7,10 @@
   justify-content: space-around;
 }
 
+.card:hover {
```

Comment:
Remember that the selectors like :hover in sass can be nested inside the class like this (in case you have defined that class for something besides the :hover).

```scss
.card {
  width: 200px;
  
  &:hover {
  ...
  }
}
```